src/store/factory/client.js

```diff
@@ -39,6 +39,7 @@ import cryptoassets from '@/utils/cryptoassets'
 import buildConfig from '../../build.config'
 import { ChainNetworks } from '@/store/utils'
 import state from '../state'
+import store from '../../store'
```

Comment:
store I think is the right thing to use. 
`../state` will just return default values i think. Shouldn't use this